lib/policyBuilder.ts

```diff
@@ -1,6 +1,14 @@
-import { createMongoAbility, MongoAbility, RawRuleOf, InferSubjects } from "@casl/ability";
+import {
+  createMongoAbility,
+  MongoAbility,
+  RawRuleOf,
+  InferSubjects,
+  MongoQuery,
+} from "@casl/ability";
 import { FormRecord } from "@lib/types";
 import { User, Privilege as PrismaPrivilege } from "@prisma/client";
+import { AnyObject } from "immer/dist/internal";
```

Comment:
Instead of importing a type from a package that is a dependency elsewhere in the application, we could just clone the type directly in the file.

```js
type AnyObject = {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  [key: string]: any;
};
```

Reply:
Will change